translation-web/src/main/java/eu/europeana/api/translation/config/TranslationConfig.java

```diff
@@ -140,4 +158,30 @@ public String getConfigFolder() {
     return CONFIG_FOLDER;
   }
   
+  public String geteTranslationUsername() {
```

Comment:
please rename to follow conventions for getter and setter metods,  getETranslationUserName